packages/shared/src/styles/themes.scss

```diff
@@ -290,5 +292,7 @@
         --badge-blue: #{$color-blue-4};
         --badge-violet: #{$color-blue-2};
         --badge-green: #{$color-green-3};
+        // wallets
+        --wallet-demo-bg-color: #fff8f9;
```

Comment:
define it in the `packages/shared/src/styles/constants.scss` file and use it here